<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ambient</class>
 <widget class="QWidget" name="Ambient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>482</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <item row="2" column="0">
    <widget class="QLabel" name="mode_label">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="mode"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="screen_label">
     <property name="text">
      <string>Screen</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QTooltipedSlider" name="framerate">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="framerate_label">
     <property name="text">
      <string>Framerate</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QFrame" name="crop_frame">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="top_label">
        <property name="text">
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="width_label">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="left">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="height_label">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="width">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="left_label">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="top">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="height">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QPushButton" name="select_rectangle">
        <property name="text">
         <string>Select rectangle...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="select_screen">
     <property name="text">
      <string>Select screen</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTooltipedSlider" name="smoothness">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="screen"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="smoothness_label">
     <property name="text">
      <string>Smoothness</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="crop_stream">
     <property name="text">
      <string>Crop stream</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTooltipedSlider</class>
   <extends>QSlider</extends>
   <header location="global">QTooltipedSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
